rotkehlchen/globaldb/handler.py

```diff
@@ -1347,6 +1349,36 @@ def soft_reset_assets_list() -> Tuple[bool, str]:
         cursor.execute(detach_database)
         return True, ''
 
+    def save_binance_pairs(
+        self,
+        new_pairs: Iterable['BinancePair'],
+        location: Location,
+    ) -> bool:
+        query = 'INSERT OR IGNORE INTO binance_pairs(pair, base_asset, quote_asset, location) VALUES (?, ?, ?, ?)'  # noqa: E501
+        cursor = self._conn.cursor()
+        try:
+            cursor.executemany(query, [pair.serialize_for_db() for pair in new_pairs])
+            self.add_setting_value(name=f'binance_pairs_queried_at_{location}', value=ts_now())
+            return True
+        except sqlite3.IntegrityError as e:
+            log.error(f'Failed to store binance pairs in database. {str(e)}')
+        return False
+
+    def get_binance_pairs(self, location: Location) -> List['BinancePair']:
+        from rotkehlchen.exchanges.utils import \
+            BinancePair  # noqa: E501  # pylint: disable=import-outside-toplevel
```

Comment:
Hmm this is indeed tricky to get right due to the way you put `BinancePair` also contain decoded assets which required the GlobalDB. Woudl need bigger refactoring and thinking on how to get rid of it. Please add a TODO, to change this.

Also look at my suggestion. Don't let isort split the line like this.

```suggestion
        from rotkehlchen.exchanges.utils import BinancePair  # noqa: E501  # pylint: disable=import-outside-toplevel  # isort:skip
```